Leetcode/tree/binary_tree.hpp

```diff
@@ -0,0 +1,39 @@
+//
+//  binary_tree.hpp
+//  tree
+//
+//  Created by dsh on 2021/7/21.
+//
+
+#ifndef TREE_BINARY_TREE
```

Comment:
```suggestion
#ifndef TREE_BINARY_TREE_HPP
```

https://google.github.io/styleguide/cppguide.html#The__define_Guard